bin/p2-run-hooks/main.go

```diff
@@ -34,7 +35,20 @@ func main() {
 		*nodeName = hostname
 	}
 
-	dir := hooks.NewContext(*hookDir, *podRoot, &logging.DefaultLogger)
+	var auditLogger hooks.AuditLogger
+	auditLogger = hooks.NewFileAuditLogger(&logging.DefaultLogger)
+	if *sqlitePath != "" {
+		al, err := hooks.NewSQLiteAuditLogger(*sqlitePath, &logging.DefaultLogger)
+		if err != nil {
+			logging.DefaultLogger.Errorf("Unable to connect sqlite database at %s, printing audit logs to STDOUT %v", *sqlitePath, err)
+		}
+		auditLogger = al
+	}
+	if auditLogger == nil {
+		log.Fatalf("DEBUG: audit logger not available")
```

Comment:
don't need `f`. also is this if statement necessary? would only happen if one of the constructors returned a `nil` reference and also `nil` error